### DISCUSSION TITLE
Combat Changes - Power Attack Like Feats

### DISCUSSION THREAD

**Discussion by Dany-Deaneris** [Rughalt/D35E#664] [Q&A] [answered]
Hello everyone, 
i want to create a feat that will work a little like power attack.  
Rughalt told me to try to use combat changes and sliders mechanic.  i've look into it and i'm a little lost.  I though i could use this thread to keep track of how this work for everyone.

For now i would like to know how to make the same effect of power attack with those mechanic.
I want a bar that we can use the bab to add to damage.  
And if even someone have more time to give new idea of what we could do with those mechanic and how that would be nice so we could create new feats.

**Comment by Rughalt** [comment, marked as answer]
Its quite easy - you need to make a feat that has following combat changes
![image](https://user-images.githubusercontent.com/802214/125337204-ff976c00-e34e-11eb-81bf-818961ed762c.png)
This part enables the slider and makes the maximum value of a slider to be BAB
![image](https://user-images.githubusercontent.com/802214/125337262-1211a580-e34f-11eb-92c6-9f3e31a785b3.png)
Here you add the selected value to damage (using `@range`) and deduct it from the attack roll!

I have attached the feat to the ticket
[fvtt-Item-its-like-power-attack!.json.zip](https://github.com/Rughalt/D35E/files/6803585/fvtt-Item-its-like-power-attack.json.zip)

**Reply by Dany-Deaneris** [reply]
Nice worked fine.
